### ISSUE TITLE
low mIOU in train with coco and ResNet as backbone

### ISSUE THREAD

**Issue by zwtburger** [jfzhang95/pytorch-deeplab-xception#65]
hi, thank you for your code, I trained the network use coco dataset and ResNet as backbone with 40 epoches, the problem is that Val mIOUs are same value and low (about 0.03x). I just run train_coco.py, are there any wrong on my work? Thanks.

**Comment by guanghuixu** [comment]
I met the same problem， do you have any idea?

**Comment by dfan** [comment]
I believe the issue is with the way images are padded in the [RandomScaleCrop](https://github.com/jfzhang95/pytorch-deeplab-xception/blob/master/dataloaders/custom_transforms.py#L88) data augmentation. As is, the code pads with 0s, but 0 is one of the MS COCO classes evaluated on. You should pad with say 255, and ignore 255 when calculating the cross entropy loss. Also, I believe the starting learning rate of 0.01 is too high - try 0.001. Hope this helps

**Comment by Qirui-Y** [comment]
@dfan  Thanks for your plan, I also encountered the same problem, can you provide more detailed ?
